<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hestia</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabScenario">
       <attribute name="title">
        <string>Scenario</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitterScenario">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTreeView" name="treeViewScenario"/>
          <widget class="QTextEdit" name="textEditScenario"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabStructure">
       <attribute name="title">
        <string>Structure</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitterStructure">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTreeView" name="treeView"/>
          <widget class="QListView" name="listView"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuScenario">
    <property name="title">
     <string>Scenario</string>
    </property>
    <addaction name="actionScenarioNew"/>
    <addaction name="actionScenarioSave"/>
    <addaction name="actionScenarioSaveAs"/>
    <addaction name="actionScenarioRemove"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuScenario"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionScenarioNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionScenarioSaveAs">
   <property name="text">
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionScenarioRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionScenarioSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
